compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -1206,10 +1203,10 @@ def IREELinalgExt_WinogradFilterTransformOp : IREELinalgExt_Op<"winograd.filter_
       return getDpsInitOperand(0)->get();
     }
     ShapedType getInputType() {
-      return input().getType().cast<ShapedType>();
+      return cast<ShapedType>(input().getType());
```

Comment:
FYI for others, these cast changes are coming from https://discourse.llvm.org/t/preferred-casting-style-going-forward/68443, which was also advertised on https://mlir.llvm.org/deprecation/ (does anyone have the actual commit that prompted this change?)

Reply:
https://github.com/llvm/llvm-project/commit/57b267905110967860f35a3e97caf3260c681e6d